cli/cmd/root.go

```diff
@@ -43,8 +43,20 @@ var (
 	// These regexs are not as strict as they could be, but are a quick and dirty
 	// sanity check against illegal characters.
 	alphaNumDash              = regexp.MustCompile("^[a-zA-Z0-9-]+$")
-	alphaNumDashDot           = regexp.MustCompile("^[\\.a-zA-Z0-9-]+$")
-	alphaNumDashDotSlashColon = regexp.MustCompile("^[\\./a-zA-Z0-9-:]+$")
+	alphaNumDashDot           = regexp.MustCompile(`^[\.a-zA-Z0-9-]+$`)
+	alphaNumDashDotSlashColon = regexp.MustCompile(`^[\./a-zA-Z0-9-:]+$`)
```

Comment:
curious: why the backticks on `alphaNumDashDot` and `alphaNumDashDotSlashColon` but not on `alphaNumDash`?

Reply:
@siggy backticks relieve us from having to escape bakslashes, so we don't end up with `\\`, which we didn't have in `alphaNumDash`. But perhaps as a best practice we should always use backticks in all regexes.